build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -6,52 +6,11 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="xunit" Version="$(xunitVersion)" IsImplicitlyDefined="true" PrivateAssets="all"/>
-    <PackageReference Include="xunit.analyzers" Version="$(xunitanalyzersVersion)" IsImplicitlyDefined="true" PrivateAssets="all"/>
-    <PackageReference Include="xunit.runner.visualstudio" Version="$(xunitrunnervisualstudioVersion)" IsImplicitlyDefined="true" PrivateAssets="all"/>
-    <PackageReference Include="xunit.runner.console" Version="$(xunitrunnerconsoleVersion)" IsImplicitlyDefined="true" PrivateAssets="all"/>
-  </ItemGroup>
-
-  <!-- 
-    In order to use Test Explorer in Visual Studio as well as leverage LUT testing we need to have both of the follownig packages in 
-    our build.  
-
-        - xunit.runner.visualstudio
-        - xunit
-
-    Both of these include xunit.abstractions.dll in different forms: None and reference
-    respectively.  This creates a double write violation in our build.  The None item
-    is unnecessary for us since we reference the xunit package hence remove it here. 
-
-    https://github.com/xunit/xunit/issues/1651
-    https://github.com/dotnet/roslyn/issues/18753
-  -->
-  <ItemGroup>
-    <None Remove="$(NuGetPackageRoot)xunit.runner.visualstudio\$(xunitrunnervisualstudioVersion)\build\net20\..\_common\xunit.abstractions.dll" />
-  </ItemGroup>
-
   <PropertyGroup>
-    <!-- xUnit.net Console Runner -->
-    <XUnitPath>$(NuGetPackageRoot)\xunit.runner.console\$(xunitrunnerconsoleVersion)\tools\net452\xunit.console.x86.exe</XUnitPath>
-    <XUnitTestResultsDirectory>$(OutDir)\xUnitResults</XUnitTestResultsDirectory>
-    <XUnitArguments>"$(OutDir)\$(AssemblyName).dll" -html "$(XUnitTestResultsDirectory)\$(AssemblyName).html" -noshadow</XUnitArguments>
+    <RuntimeIdentifier Condition="'$(TargetFramework)' == 'net46' and '$(RuntimeIdentifier)' == '' and '$(OS)' == 'Windows_NT'">$(RoslynDesktopRuntimeIdentifier)</RuntimeIdentifier>
```

Comment:
I wonder if we should add `$(RoslynDesktopTargetFramework)` instead of hard coding `net46` everywhere.

Reply:
Maybe. It won't change that often and it's easy to search for `net46`, so don't think it makes much of a difference.